mp_cnn/main.py

```diff
@@ -97,7 +101,7 @@
         trainer.train(args.epochs)
 
     model = torch.load(args.model_outfile)
-    test_evaluator = MPCNNEvaluatorFactory.get_evaluator(args.dataset, model, test_loader, args.batch_size, args.cuda)
+    test_evaluator = MPCNNEvaluatorFactory.get_evaluator(dataset_cls, model, test_loader, args.batch_size, args.device)
```

Comment:
Duplication with line 80.

Reply:
It looks the same but is meant to be duplicate. This line creates a `test_evaluator` from a saved model on disk whereas the previous one is used for training.